fluss-server/src/main/java/com/alibaba/fluss/server/tablet/TabletService.java

```diff
@@ -313,6 +315,14 @@ public CompletableFuture<ListOffsetsResponse> listOffsets(ListOffsetsRequest req
     @Override
     public CompletableFuture<InitWriterResponse> initWriter(InitWriterRequest request) {
         // todo: add authorization for table acl until https://github.com/alibaba/fluss/issues/756.
+        List<PbTablePath> tablePathsList = request.getTablePathsList();
+        if (tablePathsList != null && !tablePathsList.isEmpty()) {
+            for (PbTablePath pbTablePath : tablePathsList) {
+                authorizeTable(
+                        WRITE,
+                        TablePath.of(pbTablePath.getDatabaseName(), pbTablePath.getTableName()));
+            }
+        }
```

Comment:
1. if `authorizer` is not enabled, we ignore all the table paths. 
2. if `authorizer` is enabled, only throw `AuthorizationException` if the user has no WRITE permission on all the given tables. So we can't directly use `authorizeTable` method.

Reply:
Besides, we need to add unit test for the cases in `TabletServiceITCase` or a dedicated test class:
 (1) `authorizer` is enabled, but the table path in request is empty (exception) 
 (2) `authorizer` is enabled, and 1 table has write permission, the other doesn't have permission. (pass)
 (3) `authorizer` is enabled, and no table has write permission (exception)
 (4) `authorizer` is not enabled, and the table path in request is empty (pass, the backward compatibility)